code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -16,6 +16,10 @@
 	icon_state = "ash"
 	anchored = 1
 
+/obj/effect/decal/cleanable/ash/attack_hand(mob/user as mob)
+	user.visible_message("<span class='notice'>[user] wipes away \the [src].</span>")

```

Comment:
You might want to add a message for the user as the second argument. Something like "You wipe away \the [src]" would work fine.